source/api/extended.cpp

```diff
@@ -912,16 +823,16 @@ Extended::initiateTransfer(int securitySum, const Models::Address& inputAddress,
  */
 
 Models::Address
-Extended::newAddress(const Models::Seed& seed, const int32_t& index, const int32_t& security) {
-  auto key          = Crypto::Signing::key(seed, index, security);
+Extended::newAddress(const Models::Seed& seed, const int32_t& index) {
```

Comment:
I would move this function in the Seed model and bring back the security parameter with 0 as default value.
If 0 we use the seed security, otherwise we use the seed security.
We should be able to provide 1,2 or 3 secured addresses from the same seed.